public/data/2042400-2724/info.json

```diff
@@ -0,0 +1,50 @@
+{
+    "user": {
+      "username": "Alexandra Lerma",
+      "profileImage": "https://lh3.googleusercontent.com/a/ACg8ocLQw2Ztgi6CaRswuIyRoyrb_LzCqfYM24mnpZXRkAgJJsNgz44=s288-c-no",
+      "socialLink":  "@MoonLoveless"
+    },
+    "topics": [
+      {
+        "topic": "advanced grammar",
+        "questions": [
+          {
+            "question": "Identify the correct use of a semicolon in the sentence:",
+            "options": [
+              "He loves cooking; eating and sleeping.",
+              "He loves cooking; he eats, and he sleeps.",
+              "He loves; cooking, eating, and sleeping.",
+              "He loves cooking, eating, and sleeping; he does not like cleaning."
+            ],
+            "answer": "He loves cooking, eating, and sleeping; he does not like cleaning."
+          },
+          {
+            "question": "Which sentence correctly incorporates the subjunctive mood?",
+            "options": [
+              "If I was richer, I would buy a car.",
+              "I wish she was here now.",
+              "If he were taller, he could reach the top shelf.",
+              "She will go to the park if the weather was better."
+            ],
+            "answer": "If he were taller, he could reach the top shelf."
+          }
+        ]
+      },
+      {
+        "topic": "advanced vocabulary",
```

Comment:
Not good!! The topic should be grammar or vocabulary.

Reply:
fixed